packer/files/node-exporter.service

```diff
@@ -2,7 +2,7 @@
 Description=Node Exporter
 
 [Service]
-ExecStart=/usr/bin/node_exporter --collector.interrupts
+ExecStart=/usr/bin/node_exporter
```

Comment:
Was that intentional?

Reply:
of course, I went over you comments to @benipeled PR and remove that as part of the cleanup.
This is the node_exporter that runs on the monitoring machine, we don't need interrupts collection there